internal/orchestrion/_integration/internal/trace/span.go

```diff
@@ -159,3 +160,137 @@ func printableSpanAttribute(attrs map[string]any, key string) string {
 	}
 	return fmt.Sprintf("%+v", val)
 }
+
+// FromSimplified parses a trace tree from a simplified string format, with the following criteria:
+// - 1 span per line
+// - Span format is [span.name | resource.name | component | span.kind]
+// - All values are optional except span.name
+// - Indentation (4 spaces or 1 tab) is used to represent parent-child relationships.
```

Comment:
I'm not saying don't do this... I generally am not too fond of indentation carrying semantic value.

I think this is one case where it's probably fine though: the value-carrying indentation here is not in code that'll run in production, so my typical rationale for not linking it (besides it being harder to process by folks with astigmatism or certain forms of dyslexia)  does not apply here.

Reply:
Second - Go's indentation style is tab-based. The repo uses tabs everywhere... Who even allow the 4-space format? And if this is because it's in string literals... then why even allow the tab format? 

I mean... choose one side, don't allow both.